pydis_site/apps/content/models/tag.py

```diff
@@ -0,0 +1,75 @@
+import collections.abc
+import json
+
+from django.db import models
+
+
+class Commit(models.Model):
+    """A git commit."""
+
+    URL_BASE = "https://github.com/python-discord/bot/commit/"
+
+    sha = models.CharField(
+        help_text="The SHA hash of this commit.",
+        primary_key=True,
+        max_length=40,
+    )
+    message = models.TextField(help_text="The commit message.")
+    date = models.DateTimeField(help_text="The date and time the commit was created.")
+    author = models.TextField(help_text="The person(s) who created the commit.")
```

Comment:
I called it author to follow the naming scheme of the API object, but it probably makes more sense to properly title it. In terms of the documentation, there's also the option of making the author another model, though it really only carries two important bits of info. Another option is to encode the author as a single string from the get-go if we wanted to avoid converting to and from JSON.

Reply:
There's also the option of just not caring, it's unusual for the commit author to not be the committer as well, so it's not very important info to display.

Reply:
yea I don't think we need to go as far as another model, I was just surprised when reviewing that a field titled `author` was actually a serialized json structure containing multiple authors with additional info.

Reply:
2343f7ab